src/client.ts

```diff
@@ -141,7 +141,13 @@ export default class Client {
   }
 
   private async gemNotInstalled(): Promise<boolean> {
-    const bundlerCheck = await this.execInPath("bundle check");
+    let bundlerCheck: string;
+
+    try {
+      bundlerCheck = await this.execInPath("bundle check");
+    } catch (error) {
```

Comment:
If we are not going to use the `error`, I suggest:
```suggestion
    } catch {
```
instead.